requirements.txt

```diff
@@ -1,2 +1,10 @@
+openai~=1.101.0
+falkordb~=1.2.0
+tqdm~=4.66.1
+pydantic~=2.11.7
+numpy~=1.24.4
```

Comment:
don't we have pyproj and uv? Why do we need this?

Reply:
Agree. I have [deleted](https://github.com/MartinLeitgab/AISafetyIntervention_LiteratureExtraction/pull/51/commits/eabb45e943dfc65d6c9ce944b4e4bfa2e1eb918b) requirements.